reminder/reminder.go

```diff
@@ -196,7 +196,9 @@ func (c *InstallationClient) checkReminders(ctx context.Context, issue *issue) e
 
 	check := func(author, body string) error {
 		for _, reminder := range findTimes("reminder", body) {
-			if diff := time.Until(reminder); diff > 24*time.Hour || diff < -24*time.Hour {
+			now := time.Now()
+			// not the same day
+			if reminder.Day() != now.Day() || reminder.Month() != now.Month() || reminder.Year() != now.Year() {
```

Comment:
I'm not sure about this change, we not only care about the date, but also the time zone, right?

Reply:
Makes sense. Few lines above at L192 the reminder date is constructed in UTC TZ
```go
date = time.Date(date.Year(), date.Month(), date.Day(), 0, 0, 0, 0, time.UTC)
```
Looks to me like we can change the timezone of `now` on L199 to 
```go
now := time.Now().In(time.UTC)
```
Does that make sense?